tin/apps/assignments/tasks.py

```diff
@@ -42,10 +41,11 @@ def run_moss(moss_result_id):
         published_submission = publishes.latest().submission if publishes else latest_submission
         if published_submission is not None:
             file_with_header = published_submission.file_text_with_header
-            with open(os.path.join(download_folder, f"{student.username}.{extension}"), "w") as f:
-                f.write(file_with_header)
+            assert isinstance(file_with_header, str)
```

Comment:
What's the `assert` for?

Reply:
I believe it's to make sure we're writing a string to a file (to make type checkers happy).
We don't really need it, I'll replace it with a `str()` call in the `write_text` area instead (to be safe).

Reply:
Yep - I just get scared when assert statements show up in production code - `str()` is definitely the better way to do it